src/components/nav/nav-item-dropdown.js

```diff
@@ -41,23 +47,29 @@ export const BNavItemDropdown = /*#__PURE__*/ Vue.extend({
     }
   },
   render(h) {
+    const { buttonId, visible } = this
+
     const $button = h(
       BLink,
       {
-        ref: 'toggle',
         staticClass: 'nav-link dropdown-toggle',
         class: this.toggleClasses,
-        props: { disabled: this.disabled },
+        props: {
+          href: this.href || `#${buttonId}`,
```

Comment:
I still don't think we should provide an `href` prop, as people may want to try and add `/some/path/here`, when it should only be a hashed based URL (i.e. `#somehash`)

Reply:
And having `href` work differently here than other places where it is normally used will add to user confusion.